_includes/footer.html

```diff
@@ -2,9 +2,16 @@
     <div class="container">
       <a href="mailto:abtech@andrew.cmu.edu" class="user-select-all">abtech@andrew.cmu.edu</a> |
       <a href="tel:+14122682104" class="user-select-all">412.268.2104</a> |
+      <a href="https://tartanconnect.cmu.edu/feeds?type=club&type_id=68380&tab=about" target="_blank">TartanConnect</a>
```

Comment:
Please add a vertical bar after this entry like the others to match the formatting.

Reply:
Additionally, how does this look on mobile? Be mindful of how wide the line will be and if it may spill into the next line. You may choose to break the line somewhere depending on how it looks or move things around.